### DISCUSSION TITLE
Automated Fuzzing

### DISCUSSION THREAD

**Discussion by 4thZodiac** [IntelLabs/kAFL#301] [Q&A]
Hello!

I'm a beginner working on a project that focuses on automated fuzzing. The goal is to fuzz drivers and determine whether they cause crashes, and helping to access if they're vulnerable. I'd love to hear your thoughts and opinions on whether this is feasible. Is it possible to create a harness that can fuzz any driver?

**Comment by Wenzel** [comment]
Hi @4thZodiac 
Thanks for your question, i'll try to get some time for it in about 2 weeks !
